src/data-layer-object.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * Object within the `window.dataLayer`.
+ * Object within the `window[pluginOptions[dataLayerName]]`.
```

Comment:
I think this change is not really relevant.
We should only document necessary parts with that custom change.